Cargo.toml

```diff
@@ -11,6 +11,9 @@ keywords = ["bitcoin", "wallet", "descriptor", "psbt", "taproot"]
 readme = "README.md"
 license = "MIT"
 
+[patch.crates-io]
+bdk = { git = "https://github.com/bitcoindevkit/bdk", tag = "v0.27.1", default-features = false, features = ["all-keys"] }
+
```

Comment:
It seems this patch isn't required anymore as `v0.27.1` is released and cargo will take the latest by default? So the cargo.lock update seems like only required change.

Reply:
oh sorry.. I fixed it locally but forgot to push it ! should be good to go now.